workers/unity/Assets/Playground/Scripts/MonoBehaviours/WorldCommandHelper.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+
```

Comment:
This isn't in a namespace

Reply:
Placing within `Playground.MonoBehaviours`

Reply:
Same for the others